app/kafka/manager/ActorModel.scala

```diff
@@ -59,6 +59,10 @@ object ActorModel {
                                   partitions: Int,
                                   partitionReplicaList: Map[Int, Seq[Int]],
                                   readVersion: Int) extends CommandRequest
+  case class CMAddMultipleTopicPartitions(topicsAndReplicas: Seq[(String, Map[Int, Seq[Int]])],
```

Comment:
oh I think it got deleted when I tried to rebase the commit against ac71562. Sorry I will add that back in.